content/20.treatments.md

```diff
@@ -500,14 +493,25 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of interferons.
 
+### Vaccines 
+
+#### Strategies for and challenges to vaccine development
 
-### Vaccines
+Generally, the first step in producing a vaccine is characterizing the target.
```

Comment:
'target' is an ambiguous term here. also, if you mean a gene target, this is not true historically. e.g. small pox.